.pre-commit-config.yaml

```diff
@@ -1,17 +1,16 @@
 repos:
-  - repo: https://bitbucket.org/lindenlab/git-hooks.git 
-    rev: v1.0.0-beta2
+  - repo: https://github.com/secondlife/git-hooks.git 
+    rev: v1.0.0
     hooks:
       - id: opensource-license
-      - id: jira-issue 
```

Comment:
I've explicitly removed the jira-issue git hook because it would be a huge barrier to incorporating opensource contributions.

Reply:
@vir-linden , do we still want to require that check for Linden commits?

@bennettgoble , could we make that conditional for local Linden commits if desired?

Reply:
That may be possible through some additional internal-developer-only step. Nothing prevents developers from adding their own git hooks in addition to open source friendly ones.

I'd like to suggest we merge this PR as is and follow up with additional tooling or guidance, if desired, for internal devs.